satpy/Combined_colormaps.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>I don't believe the quotes at the end of the <code>colorize</code>  line are needed. This was added as a misunderstanding by the core devs back when we switched to YAML.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>